test-requirements.txt

```diff
@@ -17,3 +17,4 @@ pytest-xdist~=1.15
 requests-mock~=1.2
 pip!=9.0.2, !=10.0.*
 wheel~=0.29
+ruamel.yaml==0.15.38
```

Comment:
Use `~=` as requested at https://github.com/coala/coala-bears/pull/2446#discussion_r195812289

Reply:
Oh, and since we've locked down test-requirements, you will need to add a new wrapper template like https://github.com/coala/coala/pull/5583

Reply:
I did not get you, can you explain this a bit more?

Reply:
No. See the PR i linked. It shows you what to do.